mobx_codegen/lib/mobx_codegen.dart

```diff
@@ -2,4 +2,4 @@ library mobx_codegen;
 
 export 'src/mobx_codegen_base.dart';
 
-const version = '0.3.13';
+const version = '0.4.0';
```

Comment:
Hope you used the `tool/expose_version.dart` script to update this. We can certainly do manually but the script just automates it

Reply:
I didn't, but will in the future. :)